rosa_cluster_admin/rosa_monitoring/rosa-understanding-the-monitoring-stack.adoc

```diff
@@ -24,8 +24,8 @@ include::modules/osd-monitoring-targets-for-user-defined-projects.adoc[leveloffs
 //Because this is a duplicate file only referenced in the ROSA topic map, the xref links in this section will differ from the osd-understanding-the-monitoring-stack.adoc file.
 
 * xref:../../rosa_cluster_admin/rosa_monitoring/rosa-accessing-monitoring-for-user-defined-projects.adoc#rosa-accessing-monitoring-for-user-defined-projects[Accessing monitoring for user-defined projects]
-* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/monitoring/understanding-the-monitoring-stack#default-monitoring-components_understanding-the-monitoring-stack[Default monitoring components]
-* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/monitoring/understanding-the-monitoring-stack#default-monitoring-targets_understanding-the-monitoring-stack[Default monitoring targets]
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/monitoring/index#default-monitoring-components_monitoring-overview[Default monitoring components]
```

Comment:
How come we're linking and not xreffing? Just wondering!

Reply:
Adding answer here too, because this is a good question: These links are from ROSA to OCP, and even though the content is in the same repo the builds don't seem to support xrefs between topic maps.